### DISCUSSION TITLE
Alertmanager flushing alerts

### DISCUSSION THREAD

**Discussion by arunkumar-sdworx** [prometheus/alertmanager#3914] [General]
alertmanager which was working fine until last month.but recently it seems flusing the alerts immedialy. what should i do to investigate this .

 the debug logs in alertmanager are below 

ts=2024-07-02T08:01:30.667Z caller=dispatch.go:164 level=debug component=dispatcher msg="Received alert" alert=api_performance_degradation_5[1f47c7b][active]
ts=2024-07-02T08:01:30.667Z caller=dispatch.go:164 level=debug component=dispatcher msg="Received alert" alert=api_performance_degradation_5[3c55bfd][active]
ts=2024-07-02T08:01:30.668Z caller=dispatch.go:516 level=debug component=dispatcher aggrGroup="{}:{alertname=\"api_performance_degradation_5\"}" msg=flushing alerts="[api_performance_degradation_5[3c55bfd][active] api_performance_degradation_5[1f47c7b][active]]"

the configuration for alertmanager 

  group_by: ["alertname"]
  group_interval: 5m
  repeat_interval: 3h
  group_wait: 60s
  receiver: blackhole

receivers:
  - name: "blackhole"

**Comment by grobinson-grafana** [comment]
Alertmanager will do an immediate flush following a restart if a flush should have happened in the past (see https://github.com/prometheus/alertmanager/blob/main/dispatch/dispatch.go#L493).

**Comment by arunkumar-sdworx** [comment]
@grobinson-grafana  Thanks ! 
After some debug i found there was some time difference between in our alerting manager container which runs in a different docker swarm. after adjusting the time it started working fine
